docs/design/3_1_1_video_capture.tex

```diff
@@ -15,10 +15,10 @@ \subsubsection{MIPI Interface and Libraries}
 The MIPI protocol has been widely adopted for image and video transmission, and as such the camera can easily be replaced by the client in the future, if desired (fulfilling constraint \textbf{C.EX.5}).
 
 % {Camera libraries or getting the video to a 'viewable' state}
-Live video streaming between the base station and the computing platform is achieved through the implementation of a Python script to stream video to a web interface. The script used to achieve this communication utilizes the Python package 'picamera,' which provides a modular Python interface to the Raspberry Pi Camera.
+Live video acquisition is managed through the utilization of the open-source 'raspicam' C++ library. Facilitating image/video capture, 'raspicam' was selected over the native Raspberry Pi camera libraries in order to perform capture and processing in a low-level C++ environment. This allows for required low-level memory manipulation/image format conversions to be performed directly on the (more powerful) PMB, rather than the PLB.
 
 % {Explored Alternatives}
-\subsubsection{Explored Alternatives}
+\subsubsection{Explored Hardware Alternatives}
```

Comment:
Hardware Alternative Explorations?

Reply:
Eh, I don't think explorations is the right word to use here